src/Components/Blazor/testassets/StandaloneApp/Pages/Counter.razor

```diff
@@ -1,8 +1,13 @@
-﻿@page "/counter"
+@page "/counter"
 
 <h1>Counter</h1>
 
-<p>Current count: @currentCount</p>
+<p>Current count:
+    @currentCount
+    It
+    is
+    Wednesday my dudes
```

Comment:
I'm guessing... this is not really going in? Probably you were experimenting with linebreaks for @ajaybhargavb's work.

Reply:
Yeah this is leftover from other stuff. Its obviously not right because it's not wednesday anymore